client/hatch.py

```diff
@@ -0,0 +1,35 @@
+import os
+import pathlib
+import subprocess
+from typing import Mapping
+
+
+def build_libwandb(
+    go_binary: pathlib.Path,
+    output_path: pathlib.PurePath,
+):
+    output_flags = ["-o", str(".." / output_path)]
+    build_mode_flags = ["-buildmode", "c-shared"]
+    vendor_flags = ["-mod=mod"]
+
+    subprocess.check_call(
+        [
+            str(go_binary),
+            "build",
+            *output_flags,
+            *build_mode_flags,
+            *vendor_flags,
+            str(pathlib.Path("cmd", "client", "main.go")),
+        ],
+        cwd="./client",
```

Comment:
maybe just

```suggestion
        cwd="client",
```

not sure if `./client` is guaranteed to work on Windows

Reply:
works for core, but rm'ed anyways, it's better, you're right.